owners/src/owners_tree.js

```diff
@@ -156,12 +180,9 @@ class OwnersTree {
    * @return {boolean} true if the user is an owner of the file.
    */
   fileHasOwner(filename, username) {
-    const allRules = this.atPath(filename).allRules;
-    const fileRules = allRules.filter(rule => rule.matchesFile(filename));
-
-    return fileRules.some(rule =>
-      rule.owners.some(owner => owner.includes(username))
-    );
+    return this.atPath(filename)
+      .fileRules(filename)
```

Comment:
I'm guessing not, but can you use `this.fileOwners(filename)` instead?

Reply:
You're right, I can! Done.